swagger_to/swagger.py

```diff
@@ -327,6 +356,8 @@ def construct_mapping(loader, node):
     swagger.base_path = adict.get('basePath', '')
 
     for path_id, path_dict in adict.get('paths', collections.OrderedDict()).items():
+        if path_id.startswith('__'):
```

Comment:
When you remove `__line__` from the mapping `items`, you can then also remove these checks.